From a659e5c9e94963ee776caefbccb438f87a637cbd Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 10 Nov 2019 13:50:22 +0200
Subject: [PATCH] [SW-6739] elpsoft: add plugin flag in Makefile to ar

This needed since ar fails to find lto symbols of libsb.a

Change-Id: I3d0d0fe956b882dde5bd7ae127f5e5b75ecf2bb5
---

diff --git a/sdk/elpsoft/makefile b/sdk/elpsoft/makefile
index 689de26..5db3129 100644
--- a/sdk/elpsoft/makefile
+++ b/sdk/elpsoft/makefile
@@ -295,7 +295,7 @@
 else
 all: ${LIBDIR}libellipsys.a
 ${LIBDIR}libellipsys.a: ${addprefix ${OBJDIR},${OBJECTS}}
-	$(AR) rs $@ ${addprefix ${OBJDIR},${OBJECTS}}
+	$(AR) rs $(LTO_PLUGIN) $@ ${addprefix ${OBJDIR},${OBJECTS}}
 	$(RANLIB) $@
 endif
 
